src/components/issue.jsx

```diff
@@ -534,7 +531,7 @@ let Issue = React.createClass({
       return (<span>Maybe moving Issue...</span>);
     } else if (SettingsStore.getShowSimpleList()){
       node = (
-        <IssueSimple card={card} isDragging={isDragging}/>
+        <IssueSimple card={card} isDragging={isDragging} onDragStart={onDragStart}/>
```

Comment:
Should be `this.onDragStart`

There's an error, when toggling simple view of issue